packages/varlock/install.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/sh
+
+# NOTE - https://varlock.dev/install.sh redirects to this file on github
+# so can be used to install by running `curl -sSfL https://varlock.dev/install.sh | sh -s`
+
+set -e
+
+GITHUB_URL="https://github.com/dmno-dev/varlock"
+GITHUB_RELEASES_URL="${GITHUB_URL}/releases"
+HOMEBREW_FORMULA_URL="https://raw.githubusercontent.com/dmno-dev/homebrew-tap/refs/heads/main/Formula/varlock.rb"
+
+OS=""
+ARCH=""
+VERSION=""
+LATEST_VERSION=""
+
+usage() {
+    require_cmd cat
+    this=$1
+    cat 1>&2 <<EOF
+$this: download + install binary for varlock
+
+USAGE:
+    $this [FLAGS] [OPTIONS] <tag>
+
+FLAGS:
+    -h, --help      Prints help information
+
+OPTIONS:
+    -b, --bindir <DIR_PATH>     Sets bindir or installation directory. Defaults to ./bin
+
+ARGS:
+    <version>       is a specific varlock version from ${GITHUB_RELEASES_URL}. (defaults to latest)
+EOF
+    exit 2
+}
+
+parse_args() {
+  BINDIR=${BINDIR:-./bin}
```

Comment:
I think this should be an absolute path, otherwise it's just going to create a /bin wherever the install script is run